lib/src/data/simulations.dart

```diff
@@ -14,6 +14,13 @@ class Simulations with ChangeNotifier {
   final _physics = [];
   final _chemistry = [];
   var prefs;
+  final _searchTags = {
```

Comment:
From the search tags, if I enter two words which are not adjacent, the search yields `zero` results.
For eg:
- `toothpick` `pattern` yields the `Toothpick Pattern` simulation
- `toothpick` `algorithm` doesn't yield any results

Reply:
I guess I don't want to change this because search doesn't needs to be very definitive right now. We will come up with a  better data management sooner or later so let's keep it this way. If you might have a better regex, I can update that